testdata/modelldcatno_Test2FlereEksempler.ttl

```diff
@@ -0,0 +1,491 @@
+
+digs:Katalog  a        owl:NamedIndividual , dcat:Catalog ;
+        dct:description         "Digdirs modelleksempler"@nb ;
+        dct:identifier          "https://raw.githubusercontent.com/Informasjonsforvaltning/model-publisher/master/src/model/model-catalog.ttl#Katalog"^^xsd:string ;
+        dct:publisher           digs:Utgiver ;
+        dct:license             <http://creativecommons.org/licenses/by/4.0/deed.no> ;
+        dct:title               "Digitaliseringsdirektoratets modellkatalog"@nb ;
+        modelldcatno:model      digs:AltMuligModell , digs:Diversemodell ;
+        dct:modified            "2017-09-28T00:00:00+01:00"^^xsd:dateTime ;
+        dct:spatial             <http://publications.europa.eu/resource/authority/country/NOR> .
+
+digs:AlternativModellformat a owl:NamedIndividual , <https://informasjonsforvaltning.github.io/modelldcat-ap-no/images/ModellDCAT-ap-no.png> , foaf:Document ;
+      dct:title                 "Forenklet modell for ModellDCAT-AP-NO"@nb ;
+      dct:format                <http://publications.europa.eu/resource/authority/file-type/PNG> .
+
+
+digs:Utgiver  a       owl:NamedIndividual , foaf:Agent ;
+        dct:identifier  "991825827"^^xsd:string ;
+        dct:type        "Organisasjonsledd"@nb ;
+        foaf:name       "Digitaliseringsdirektoratet"@nb .
+
+digs:Tidsintervall  a  owl:NamedIndividual , dct:PeriodOfTime ;
+        dcat:startDate  "2016-02-11T00:00:00+01:00"^^xsd:dateTime ;
+        dcat:endDate  "2030-02-11T00:00:00+01:00"^^xsd:dateTime .
+
+digs:KontaktOss  a        vcard:Kind , owl:NamedIndividual ;
+        vcard:fn            "Digitalisation department"@en , "Avdeling for digitalisering"@nn , "Avdeling for digitalisering"@nb ;
+        vcard:hasEmail      <mailto:informasjonsforvaltning@digdir.no> ;
+        vcard:hasTelephone  <tel:12345678> .
+
+
+ex-code:Aktør  a                      owl:NamedIndividual , modelldcatno:modelldcatno:RootObjectType ;
+        modelldcatno:hasProperty       ex-sch:hustandEierKjaeledyr , ex-mch:godteri , ex-code:kommuneliste , ex-note:merknad , ex-valgfri:utgifterBarnepass , ex-specialization:katteeksempel , ex-collection:fruktkurveksempel , ex-composition:griseeksempel , ex-abstrakt:abstraksjonseksempel , ex-rel:realisereksempel ;
+        dct:title                      "Aktør"@nb ;
+        modelldcatno:belongsToModule   "Aktør"@nb .
+
+#Eksempel ex-code beskriver hvordan man beskriver kodelister. Det er laget trådskisse på dette.
+
+ex-code:kommuneliste  a               owl:NamedIndividual , modelldcatno:Attribute  ;
+        dct:title                   "kommunelisteeksempel"@nb ;
+        modelldcatno:hasValueFrom    ex-code:kodeliste ;
+        xsd:maxOccurs             "1"^^xsd:nonNegativeInteger .
+
+ex-code:kodeliste a        owl:NamedIndividual , modelldcatno:CodeList ;
+        dct:title "Kommunenummerlisten"@nb .
+
+ex-code:kodeLillehammer      a        owl:NamedIndividual , modelldcatno:CodeElement ;
```

Comment:
Slik høsteren oppfører seg nå så vil ikke kodeLillehammer og kodeHamar bli plukket opp. 

Det blir veldig tungt for høsteren om den må tråle gjennom rdf-dataen etter eventuelle skos:inScheme, det er mye enkelere om modelldcatno:CodeList lister opp innholdet selv

Reply:
Skal nok klare å få til noe som ikke er så tungt som å alltid tråle etter skos:inScheme, jeg får vel bare lage et issue på det og vri hodet rundt det...